docs/odata/naming.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Naming Conventions
+name: naming
+parent: oData
+nav_order: 3
+---
+
+## Naming Conventions
+
+| oData Term      | Rule | Good Example | Bad Example |
```

Comment:
for juxtaposing good/bad, @marianfoo has a nice visual thing worked out: https://1dsag.github.io/UI5-Best-Practice/i18n/#good-example
might be also applicable here, at least coloring the column bg, so comprehension of what's considered good and what bad gets easier for the reader